server.html

```diff
@@ -20,13 +20,14 @@ <h1 id="server-side">I want to add CORS support to my server</h1>
 			<li><a href="server_apache.html">Apache</a></li>
 			<li><a href="server_appengine.html">App Engine</a></li>
 			<li><a href="server_aspnet.html">ASP.NET</a></li>
-			<li><a href="server_awsapigateway.html">AWS API Gateway</a></li>			
+			<li><a href="server_awsapigateway.html">AWS API Gateway</a></li>
 			<li><a href="server_caddy.html">Caddy</a></li>
 			<li><a href="server_cgi.html">CGI Scripts</a></li>
 			<li><a href="server_expressjs.html">ExpressJS</a></li>
 			<li><a href="server_iis6.html">IIS6</a></li>
 			<li><a href="server_iis7.html">IIS7</a></li>
-      			<li><a href="server_meteor.html">Meteor</a></li>
+			<li><a href="server_spring-boot_kotlin.html">Spring Boot Applications in Kotlin</a></li>
```

Comment:
Can you alphabetize this, so put "Spring Boot..." after "PHP" (I realize that Coldfusion is not alphabetized, and I will take care of that in a different change, unless you want to also do that here).